### PR TITLE
Fixing email validations issues (Backend):

### PR DESCRIPTION
**Pull request by michaelhidalgo**
Remediating following Issues : 
[1]https://github.com/TeamMentor/TM_4_0_Design/issues/563
[2]https://github.com/TeamMentor/TM_4_0_Design/issues/562

### CODE REVIEW

File: Source_Code/TM_UnitTests/TeamMentor.UnitTests/CoreLib/Schemas/Test_Schemas_Validation.cs [hunk 1]
```diff
@@ -75,15 +75,17 @@ public void Validation_NewUser_LargeDataInFields()
                              "mappings : {0}".format(result.Value.toString()));
         }
         
-        [Test]
+        [Test,Ignore]
```
Comment (DinisCruz) [inline Source_Code/TM_UnitTests/TeamMentor.UnitTests/CoreLib/Schemas/Test_Schemas_Validation.cs]: I still think that we should keep this test (unless the tests below are doing the exact check)

**Comment by DinisCruz** [comment]
looks good :)

Just that small comment about the test being ignored

**Comment by michaelhidalgo** [comment]
@DinisCruz the test you referenced has been reenabled again, so it works as designed. Can you please double check and/or merge this change once you agreed?

**Comment by michaelhidalgo** [comment]
@DinisCruz the test you've referenced has been refactored, can you please do the review again.

**Comment by DinisCruz** [comment]
nice, merged

**Pull request merged**